VideoLocker/src/main/java/org/edx/mobile/model/course/BlockType.java

```diff
@@ -4,6 +4,12 @@
  * Created by hanning on 5/21/15.
  */
 public enum  BlockType {
-    COURSE, CHAPTER,  SECTION,  SEQUENTIAL,
-    VERTICAL,  VIDEO,  HTML, PROBLEM, DISCUSSION, OTHERS
+    COURSE(false), CHAPTER(false),  SECTION(false),  SEQUENTIAL(false),
```

Comment:
so, are we going with the current solution or should I change it to the `switch-case` counterpart ?

and yeah `isContainer` method seems to be a better idea.